### PR TITLE
docs: added registry templating variable

### PR DESCRIPTION
**Pull request by skoh7645**
## Checklist

- [x] commit message follows [commit guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).
- [x] website builds and runs in production - for information on how to test locally go [here](https://github.com/appsody/website/blob/master/DEVELOPMENT.md).

## Description

Added `image-registry` to the table of available templating variables

## Related Issues
This should go in when https://github.com/appsody/appsody/pull/750/files gets merged

### CODE REVIEW

File: content/docs/stacks/modify.md [hunk 1]
```diff
@@ -75,8 +75,8 @@ The variables that stack creators can use to access stack values are:
 | `.stack.semver.minor`     | The `version` minor value from `stack.yaml`.                                          |
 | `.stack.semver.patch`     | The `version` patch value from `stack.yaml`.                                          |
 | `.stack.semver.majorminor`| The `version` major and minor values from `stack.yaml`.                               |
-| `.stack.image.namespace`  | The `image-namespace` from user defined image-namespace flag, default is `dev.local`. |
-
+| `.stack.image.namespace`  | The `image-namespace` from user defined image-namespace flag, default is `appsody`. |
```
Comment (helenmasters) [inline content/docs/stacks/modify.md]: I think that these descriptions need to be clearer.
What do we mean by "the user defined image-namespace" flag?
Do we mean something like...

 "The `image-namespace` value from the `--image-namespace` flag, if it's passed to the Appsody CLI; default is `dev.local`."   I presume that this flag can only be passed to certain CLI commands as well, are we intending to list those?

Comment (helenmasters) [reply content/docs/stacks/modify.md]: Same comment for the `stack.image.registry` description.

Comment (skoh7645) [reply content/docs/stacks/modify.md]: Templating happens on `appsody stack package`, so the flags would have to be passed to `appsody stack package`

Comment (skoh7645) [reply content/docs/stacks/modify.md]: Would "The image-namespace value from the `--image-namespace` flag that is passed to the `appsody stack package` command ; default is dev.local." be clearer? (And similarly for `stack.image.registry`

Comment (helenmasters) [reply content/docs/stacks/modify.md]: Can we shorten it again to the following...?
"The value from the `image-namespace` flag that is passed to the `appsody stack package` command; default is dev.local."

**Pull request merged**